manifests/dashboard.pp

```diff
@@ -105,4 +94,18 @@
     hasrestart => true,
     name       => $dashboard_service,
   }
+
+  exec {'Waiting for Wazuh dashboard...':
+    require => Service[$dashboard_service],
+    command => "sleep 15 ",
+    path => "/usr/bin:/bin",
+  }
+
+  file { '/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml':
+    owner   => 'wazuh-dashboard',
+    group   => 'wazuh-dashboard',
+    mode    => '0600',
+    content => template('wazuh/wazuh_yml.erb'),
+    require => Package['wazuh-dashboard']
+  }
```

Comment:
```suggestion
  file { '/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml':
    owner   => 'wazuh-dashboard',
    group   => 'wazuh-dashboard',
    mode    => '0600',
    content => template('wazuh/wazuh_yml.erb'),
    require  => Package['wazuh-dashboard'],
    before   => Service['wazuh-dashboard'],
  }
```

Reply:
It is not possible to apply this suggestion because this file is not created with the installation of the package, for this reason, the first solution was to apply a delay to give time for this file to be created.

Now we have taken a new approach, first we create the directories and then the file, in this way the file is maintained and works correctly, without the need to wait for a delay:
https://github.com/wazuh/wazuh-puppet/blob/dd7060b4975d414794c1138329702b2d1207935a/manifests/dashboard.pp#L102-L119